### ISSUE TITLE
Edit Custom Tonie => select file: Filter does not work

### ISSUE THREAD

**Issue by fila612** [toniebox-reverse-engineering/teddycloud#224]
When you add a tag file to a custom tonie, filtering is not possible.
After select a TAF file, it jumps to the table header and its not sure if the selection was correct

**Comment by henryk86** [comment]
one problem per ticket and gui related tickets in the teddycloud_web repository please.

>> When you add a tag file to a custom tonie, filtering is not possible.
I cannot reproduce that. What are you doing exactly? Filtering for the taf file name is not possible in general.
If i add a ta**f** file to a custom tonie and save and search for the title of that taf file (assume the taf file has an entry in the tonies(.custom).json), the tonies are filtered accordingly, especially the changed one is found.

>> After select a TAF file, it jumps to the table header and its not sure if the selection was correct
Please give some more details to. Cannot reproduce it with this...

**Comment by fila612** [comment]
okay, here are the steps to reproduce it:
1. select a custom tonie and click on the edit button (pencil)
2. new overlay opens to select a source and the corresponding model
3. click to select a source to the folder icon, right from the path line
4. in my case: choose a folder and the use the text filter to narrow down the results
=> no changes by tipping in some letters

When its used the text filter in the lib, its working.

the second behaviour is more or less a subsequent error:
when I realised, that the textfilter don't work: I scrolled down to the needed taf-file and click on the checkbox.
Now the list jumps back to the top of the table and I don't see if the selection was right (had to scroll down again for verification)
